### PR TITLE
Update scatterplot.Rmd

### PR DESCRIPTION
**Pull request by leahparreztnik**
I understand that there is some overlap here with the "when to use" section. I think it could be beneficial to be consistent and either have a when to use section in every "chapter" or let it be subsumed into theory. Maybe this is too knitpicky but I think there is more to be said and it would be good to have consistency.

### CODE REVIEW

**Comment by zachbogart** [comment]
Agreed. We like knitpicky. Sounds like you are on the case! Do you have a preference?

Either way, consider proposing this as an issue and linking your relevant PRs. Looks like you are on your way to adding theory to the resource. Let us know if you need assistance.

File: scatterplot.Rmd [hunk 1]
```diff
@@ -140,6 +140,11 @@ scatter +
 `ggplot2` makes it very easy to create scatterplots. Using `geom_point()`, you can easily plot two different aesthetics in one graph. It also is simple to add on extra formatting to make your plots look nice (All that is really necessary is the data, the aesthetics, and the geom).
 
 ## Theory
+
+Scatterplots are very useful in understanding the correlation (or lack thereof) between variables.  For example, in section 13.1 notice the positive relationship between brain and body weight in species of land mammals. The scatterplot gives a good idea of whether that relationship is positive or negative and if there is correlation.  Don't mistake correlation in a scatterplot for causation!
```
Comment (zachbogart) [inline scatterplot.Rmd]:
```suggestion
Scatterplots are very useful in understanding the correlation (or lack thereof) between variables.  For example, in [section 13.2](scatter.html#tldr-7) notice the positive relationship between brain and body weight in species of land mammals. The scatterplot gives a good idea of whether that relationship is positive or negative and if there's a correlation.  However, don't mistake correlation in a scatterplot for causation! 
```

Comment (zachbogart) [reply scatterplot.Rmd]: Please update with these small changes. Make sure I did the link correctly :)

Comment (leahparreztnik) [reply scatterplot.Rmd]: I think it worked.  As for the differentiation between "when to use" and "theory", I think the "when to use" can be a bit more to the point, while the theory gives the best practices and how to decide between variations in the graph.

**Comment by zachbogart** [comment]
Nicely done! Very much appreciate your attention toward the theory. It's an important section.

**Pull request merged**